python/core/additions/qgssettingsentry.py

```diff
@@ -22,15 +22,15 @@
 import qgis  # required to get base class of enums
 
 
-class _PyQgsSettingsEntryEnumFlag(QgsSettingsEntryBase):
+class PyQgsSettingsEntryEnumFlag(QgsSettingsEntryBase):
     """ class QgsSettingsEntryEnum
         ingroup core
         An enum settings entry.
         since QGIS 3.20
     """
```

Comment:
```suggestion
    """ class QgsSettingsEntryEnum
    ingroup core
    An enum settings entry.
    since QGIS 3.20
    """
```

can you fix the others docstring?